libr/include/r_util/r_utfxconv.h

```diff
@@ -0,0 +1,5 @@
+#ifndef R_UTFXCONV_H
```

Comment:
no need a new include file for just 2 definitions. we can pack them into r_utf8.h